cli/tests/test_bisect_command.rs

```diff
@@ -96,19 +104,10 @@ fn test_bisect_run() {
     ");
 
     // Try with legacy command argument
-    insta::assert_snapshot!(work_dir.run_jj(["bisect", "run", "--range=..", "--command", "false"]), @r"
-    Now evaluating: royxmykx dffaa0d4 c | c
-    The revision is bad.
-
-    Now evaluating: rlvkpnrz 7d980be7 a | a
-    The revision is bad.
-
-    Search complete. To discard any revisions created during search, run:
-      jj op restore 5473934e3b2f
-    The first bad revision is: rlvkpnrz 7d980be7 a | a
```

Comment:
I updated the test to only look at stderr in order to avoid this the op here that depends the command path that got recorded by the first `jj bisect` command.